cmd/experimental/kueue-viz/frontend/.dockerignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+npm-debug.log
+build
+.git
+.gitignore
+README.md
+.env*
+!.env.example
```

Comment:
Oh, sorry, this is about `.dockerignore`, and we don’t need `.env.example` either.

Reply:
fixed